apidoc/Titanium/UI/Window.yml

```diff
@@ -6,6 +6,21 @@ since: "0.9"
 excludes: { methods:[removeAllChildren] }
 
 methods:
+  - name: getSustainedPerformanceMode
+    summary: returns sustainedPerformanceMode property
+    returns:
+        type: Boolean
+    since: "6.1.0"
+    platforms: [android]
+
+  - name: setSustainedPerformanceMode
+    summary: sets sustainedPerformanceMode property
+    parameters:
+      - name: mode
+        summary: true to enable sustained performance mode
+        type: Boolean
+    since: "6.1.0"
+    platforms: [android]
```

Comment:
You don't need the custom getter and setter, they are auto-generated in the docs